### ISSUE TITLE
revision started from user A can be completed by user B

### ISSUE THREAD

**Issue by strk** [linz/postgresql-tableversion#179]
It is possible that a revision started by a user is completed by another user.
I'm not sure this should be the case.
NOTE: this scenario is documented as possible in the current `doc/table_version.md` file.

See also #178 

\cc @palmerj

**Comment by palmerj** [comment]
> It is possible that a revision started by a user is completed by another user.

How is this? The revision ID is stored in the temp table for the SQL user session.

**Comment by strk** [comment]
a temp table survives change in session authorization (`SET SESSION AUTHORIZATION`).
that's how I tested (changed to a different user and run complete_revision):
```
tv=# select table_version.ver_create_revision('test');
 ver_create_revision 
---------------------
                1002
(1 row)

tv=# set session_authorization to guest;
SET
tv=> select table_version.ver_complete_revision();
 ver_complete_revision 
-----------------------
 t
(1 row)
```

**Comment by strk** [comment]
I've pushed a fix for this in #181
